### ISSUE TITLE
Linker issues with "cargo build" in Windows?

### ISSUE THREAD

**Issue by mattia-donato** [first-rust-competition/ni-fpga-rs#31]
Hello,

I am trying to compile with the command cargo build your version of "ni-fpga-rs" and I get a linker error.

```
  = note: LINK : fatal error LNK1181: cannot open input file 'NiFpga.lib'
error: could not compile `integration` (bin "integration") due to previous error

```

Where I can get it? I can find only NiFpga.DLL in LabView folder.

**Comment by mattia-donato** [comment]
Thank you. Finally I could compile. I still have to testing but it is possible to convert the nifpga.dll to nifpga.lib by the following by running on the cmd.exe inside Visual Studio enviroment the following

```
echo LIBRARY NIFPGA > nifpga.def
echo EXPORTS >> nifpga.def
for /f "skip=19 tokens=4" %A in ('dumpbin /exports nifpga.dll') do echo %A >> nifpga.def
lib /def:nifpga.def /out:nifpga.lib /machine:x64
```

**Comment by ThadHouse** [comment]
This is fixed with #33. It now dynamically links to nifpga, which is installed on system32 on windows. @auscompgeek this should be closable now.

**Comment by auscompgeek** [comment]
I'll bump the versions later today.
